.bazelrc

```diff
@@ -34,6 +34,9 @@ build --tool_java_runtime_version=remotejdk_11
 # build --platform_mappings=""                                                               # unique
```

Comment:
For these lines above, since `clang-asan` no longer exists and the tests are passing, can we remove the unique lines and bring this closer to Envoy?

```
build:clang-asan --test_timeout=900                                                         # unique
build:clang-tsan --test_timeout=900                                                          # unique
```

Reply:
Ironically I think these two lines are the reason why the CI is still "passing" even though the sanitizers aren't running anymore. Envoy removed `clang-tsan` configs, but we don't get an error about missing configs even though our CI script still refers to them, because these to liones make `clang-tsan` and `clang-asan` into valid configs.

Reply:
That's how they get you! Removed those.